features/createimagebitmap.yml

```diff
@@ -1,11 +1,18 @@
 name: createImageBitmap
 description: The `createImageBitmap()` global method creates an `ImageBitmap` object from a source such as an image, SVG, blob, or canvas. An `ImageBitmap` object represents pixel data that can be drawn to a canvas with lower latency than other types, such as `ImageData`.
-caniuse: createimagebitmap
 spec: https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmap
+caniuse: createimagebitmap
+status:
+  compute_from:
+    - api.createImageBitmap.svgimageelement_as_source_image
+    - api.createImageBitmap.options_resizeQuality_parameter
+    - api.createImageBitmap.options_resizeWidth_parameter
+    - api.createImageBitmap.options_resizeHeight_parameter
 compat_features:
   - api.createImageBitmap
   - api.createImageBitmap.options_colorSpaceConversion_parameter
   - api.createImageBitmap.options_imageOrientation_parameter
+  - api.createImageBitmap.options_imageOrientation_parameter.from-image
```

Comment:
Seems like enough of a minor addition, so +1 to add it here and hide it behind compute_from.